include/mbedtls/ecdh.h

```diff
@@ -325,7 +325,7 @@ int mbedtls_ecdh_read_params(mbedtls_ecdh_context *ctx,
  * \brief           This function sets up an ECDH context from an EC key.
  *
  *                  It is used by clients and servers in place of the
- *                  ServerKeyEchange for static ECDH, and imports ECDH
+ *                  ServerKeyExchange for static ECDH, and imports ECDH
```

Comment:
The typo `ServerKeyEchange` still seems to exist in `3rdparty/everest/include/everest/everest.h` and `3rdparty/everest/include/everest/x25519.h`

Reply:
nevermind: these are now in tf-psa-crypto